core/src/epicli/data/common/ansible/playbooks/roles/opendistro_for_elasticsearch/tasks/configure-es.yml

```
@@ -47,3 +46,132 @@
     name: elasticsearch
     state: started
     enabled: yes
+
+- name: Change default users
+  when: not (context is defined and context == 'upgrade')
+  block: 
+    - name: Set helper facts
+      set_fact:
+        elasticsearch_endpoint: >-
+          https://{{ ansible_default_ipv4.address }}:9200
+      vars:
+        uri_template: &uri
+          client_cert: /etc/elasticsearch/kirk.pem
+          client_key: /etc/elasticsearch/kirk-key.pem
+          validate_certs: false
+          body_format: json
+
+    - name: Check if default admin user exists
+      uri:
+        <<: *uri
+        url: "{{ elasticsearch_endpoint }}/_opendistro/_security/api/internalusers/admin"
+        method: GET
+        status_code: [200, 404]
+      register: admin_check_response
+      until: admin_check_response is success
+      retries: 12
+      delay: 5
```

```suggestion
      retries: 60
      delay: 1
```
If big number of retries is used because Elasticsearch may be not ready here yet, you can consider adding task to wait for its port to become open (module `wait_for`).